### DISCUSSION TITLE
https://blog.3place.cn/3place/lmyn/LMYN054.html

### DISCUSSION THREAD

**Discussion by giscus** [hbaolong/hbaolong.github.io#106] [General]
# https://blog.3place.cn/3place/lmyn/LMYN054.html

https://blog.3place.cn/3place/lmyn/LMYN054.html

**Comment by picklecai** [comment]
神牛众友大斗法！

**Comment by picklecai** [comment]
我决不会把你丢掉。 （1.53.9）

正如我不能丢掉我的家，
舍波罗！你没有对不起我。
那个有力量的人正用武力，
把你从我身边抢劫掠夺。 （1.53.10）

我的力量比不上国王，
特别是因为他想要你。
国王刹帝利有力量，
他正在保卫着这大地。 （1.53.11）

他的兵卒挤满这里，
带着骏马和车辆，
还有旗帜和大象，
他才真正有力量。’ （1.53.12）

这是秀才遇到兵，修仙者也抵不过世俗国王的武力。真正的暴力机器，遇到该当如何？

**Comment by picklecai** [comment]
’人们说刹帝利没力量，
力量更大的是婆罗门，
婆罗门力量上天所授，
婆罗门！刹帝利不能并论。 （1.53.14）

你的力量无量无边，
没有人比你力量大。
毗奢蜜多罗固然有力，
你的威力他无法招架。 （1.53.15）

蔺相如表达过：匹夫之怒，可怒发冲冠，血溅当场。当人的全部潜力被激发出来后，“虽千万人吾往矣”。

**Reply by picklecai** [reply]
不要惹怒老实人，不要惹怒一群老实人，不要犯众怒，不要失去民心。

**Comment by picklecai** [comment]
我得到你那梵力的哺养，
光辉的仙人！请委托我，
那个坏蛋的傲气和力量，
我能够把它一一摧破。’ （1.53.16）

即使是一头牛！这就是“兔子急了也会咬人”。
